Process.h

```diff
@@ -254,6 +254,7 @@ typedef struct Process_ {
    unsigned int tree_right;
    unsigned int tree_depth;
    unsigned int tree_index;
+   bool isRoot; /* Has no known parent process */
```

Comment:
Insert blank line before; the comment describing the field should go before the field, cf. above.